### PR TITLE
Ml survey 4.3

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

**Comment by amorphous-1** [comment]
Hello @priyanka-TL,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://865.qa.docs.sunbird.org/4.3.0/ .<br/> **Note:** Only _4.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://865.qa.docs.sunbird.org/4.3.0/apis/ml-survey// <br /> - http://865.qa.docs.sunbird.org/4.3.0/apis/ml-survey// <br />  <br/> **Modified sections are -** <br />

File: apis/ml-survey/ml-surveyapi.yaml [hunk 1]
```diff
@@ -0,0 +1,1918 @@
...
+            type: string
+        - in: path
+          name: observationId
+          required: true
+          description: ID of the observation whose details to be viewed
+          schema:
+            type: string 
+          example: 5d286eace3cee10152de9efa
+        - in: query
+          name: entityId
```
[the remark this answers is not in the document]
Comment (priyanka-TL) [reply apis/ml-survey/ml-surveyapi.yaml:53]: in url only path parameter only need to add

File: apis/ml-survey/ml-surveyapi.yaml [hunk 2]
```diff
@@ -0,0 +1,1918 @@
...
+          'The body parameter refers to the format of the request, the request
+          body (payload) takes the following media types to make the payload:'
+           
+           - JSON 
+           
+          The body contains resource object for update observation submission  
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/observationsubmissionsapiUpdateRequest"
```
[the remark this answers is not in the document]
Comment (priyanka-TL) [reply apis/ml-survey/ml-surveyapi.yaml:513]: yes it's not mandatory

File: apis/ml-survey/ml-surveyapi.yaml [hunk 3]
```diff
@@ -0,0 +1,1918 @@
...
+          description: 'BAD REQUEST. The operation failed ! The possible reason for failure is that you may have missed providing input for a mandatory parameter.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/observations400Error'
+        
+      deprecated: false
+      security:
+        - bearer: []
+  /observations/mlsurvey/v1/searchEntities:
```
[the remark this answers is not in the document]
Comment (priyanka-TL) [reply apis/ml-survey/ml-surveyapi.yaml:90]: in url we need to add only path values

**Pull request merged**